site/content/en/docs/Advanced/_index.md

```diff
@@ -4,5 +4,3 @@ date: 2019-01-03T05:44:55Z
 description: "Advanced Guides, Techniques and walk-throughs"
 weight: 300
 ---
-
```

Comment:
Just to be clear - this is removed, as that pagelist is provided automatically in the latest version of Hugo + Docsy, so it's no longer needed.